mods/pageviews.js

```diff
@@ -251,6 +251,12 @@ var validateYearMonthDay = function(rp) {
 PJVS.prototype.pageviewsForArticleFlat = function(restbase, req) {
     var rp = req.params;
 
+    // dates are passed in as YYYYMMDD but we need the HH to match the loaded data
+    // which was originally planned at hourly resolution
+    if (rp.start && rp.end) {
```

Comment:
Instead of working around the fact that you dropped `HH` from the parameters, you should fix the validation logic, since it will still report that `HH` is needed, which will be pretty confusing for users.